conseil-common/src/test/scala/tech/cryptonomic/conseil/common/bitcoin/BitcoinFixtures.scala

```diff
@@ -0,0 +1,278 @@
+package tech.cryptonomic.conseil.common.bitcoin
+
+import java.sql.Timestamp
+
+import tech.cryptonomic.conseil.common.bitcoin.rpc.json._
+
+/**
+  * Fixtures for Bitcoin based on block at height 102000 with hash 00000000000335c47dd6ae953912d172a4d9839355f2083165043bb6f43c2f58
+  */
+trait BitcoinFixtures {
```

Comment:
It might help to provide docs for individual sub-values or objects in the trait, or some overall example.

I'm considering a new developer needed to add new tests or improve the existing ones. It might turn out that he needs to scan all the file to start figuring out what is what, and then start putting some effort into making the real changes.
WDYT?

Reply:
Check it now